assets/javascripts/application.js

```diff
@@ -45,7 +45,8 @@ var sso = require('./modules/sso.js'),
     charCounter = require('./modules/charCounter.js'),
     addRemove = require('./modules/addRemove.js'),
     modalDialog = require('./modules/modalDialog.js'),
-    dynamicGaTags = require('./modules/dynamicGaTags.js');
+    dynamicGaTags = require('./modules/dynamicGaTags.js'),
+    responsiveInlineDetails = require('./modules/responsiveInlineDetails.js');
```

Comment:
Any suggestions are welcome ... The behaviour of this (thinking mobile first) is a block details like tag that changes to an inline list at larger breakpoints